integration/src/main/java/scripts/testscripts/DatasetOperations.java

```diff
@@ -161,12 +164,26 @@ private void previewUserJourney(StorageSystem storageSystem, String sourceId)
     datasetId = null;
   }
 
+  private ObjectNode expectedMetadataResponse(ObjectNode metadata, StorageSystem storageSystem) {
+    ObjectNode expectedMetadata = metadata.deepCopy().put("id", datasetId.toString());
+    if (storageSystem.equals(StorageSystem.TDR)) {
+      expectedMetadata
+          .put("phsId", "1234")
+          .put(
+              "dcat:accessURL",
```

Comment:
I don't think this is the correct property for this attribute. This isn't the "access URL", it's the "request access URL". https://www.w3.org/TR/vocab-dcat-3/#Property:distribution_access_url.

Using the same property as accessURL would also mean that we couldn't use it to determine if the URL was supposed to be used for Request Access or not. Additionally, I think it's plausible that a dataset will have both a request access URL and an access URL, and that they'd be different.